.puppet-lint.rc

```diff
@@ -0,0 +1,5 @@
+--no-documentation-check
+--no-140chars-check
+--no-autoloader_layout-check
+--no-class_inherits_from_params_class-check
+--no-only_variable_string
```

Comment:
Can you remove this file?

Reply:
Can you disable the `only_variable_string` check? I needed to convert and integer into a string and it triggers that warning. See [DOCUMENT-405](https://tickets.puppetlabs.com/browse/DOCUMENT-405) for why this shouldn't be triggered.

Reply:
file removed.